### DISCUSSION TITLE
RGB-to-VGA adapter

### DISCUSSION THREAD

**Discussion by RichardHallas** [UzixLS/zx-sizif-512#31] [Q&A]
I'd like to connect my Sizif-512 to a VGA monitor, and my attention has recently been drawn to this very neat little adapter:

https://github.com/jeffqchen/9DIN2VGA

It looks to be just what I need, and even has a nice 3D-printed case to go with it. Unfortunately, it's presented on GitHub as a DIY project, and I just don't have the skills or equipment to build one myself.

Is there anyone here who makes and sells these adapters, or knows of anyone who does? Thanks…

**Comment by romeoandreica** [comment]
@UzixLS I am also interested in something like this because it is very hard to find SCART TVs, well almost impossible and the next best thing to have something that is 'retro' is a 4:3 VGA monitor but the problem is connecting the board to it without converters that need to be powered separately.

From what I saw on the VGA-15 there is the pinout that contains R,G,B + HSYNC/VSYNC ; would the board work with such a setup if a special cable is used?

**Reply by UzixLS** [reply]
If you have 15-khz compatible monitor supporting composite sync - why not?

**Comment by romeoandreica** [comment]
@UzixLS yes, I did also some research myself but such a monitor is not very common, but I have found here a list of the ones that do:

http://15khz.wikidot.com/

Most likely it's the old ones with SCART so it makes no sense to use a VGA cable if you can use SCART, in any case if it does not support the 15khz it won't work out of the box, this is the main issue, so to answer @RichardHallas as well, the project above is great but you need a compatible monitor to make it work.

So I think the best way to go in this case is to use the ZX-VGA-JOY as it supports multiple resolutions, refresh rates so I will go that way.
